conf/bblayers-bsp.inc

```diff
@@ -7,20 +7,10 @@ BSPLAYERS = " \
   ${OEROOT}/layers/meta-arm/meta-arm \
   ${OEROOT}/layers/meta-arm/meta-arm-toolchain \
   ${OEROOT}/layers/meta-arm/meta-arm-bsp \
-  ${OEROOT}/layers/meta-freescale \
-  ${OEROOT}/layers/meta-freescale-3rdparty \
   ${OEROOT}/layers/meta-raspberrypi \
   ${OEROOT}/layers/meta-riscv \
-  ${OEROOT}/layers/meta-rtlwifi \
   ${OEROOT}/layers/meta-intel \
   ${OEROOT}/layers/meta-yocto/meta-yocto-bsp \
-  ${OEROOT}/layers/meta-xilinx/meta-xilinx-core \
-  ${OEROOT}/layers/meta-xilinx/meta-xilinx-bsp \
-  ${OEROOT}/layers/meta-xilinx/meta-xilinx-standalone \
-  ${OEROOT}/layers/meta-xilinx-tools \
-  ${OEROOT}/layers/meta-ti \
-  ${OEROOT}/layers/meta-st-stm32mp \
   ${OEROOT}/layers/meta-tegra \
-  ${OEROOT}/layers/meta-sunxi \
   ${OEROOT}/layers/meta-lmp/meta-lmp-bsp \
```

Comment:
Can you have a commented section covering the layers that were removed here? Just to help later on when enabling them again.